external-crates/move/crates/move-cli/tests/build_tests/simple_new/args.exp

```diff
@@ -3,14 +3,74 @@ External Command `cat P1/Move.toml`:
 [package]
 name = "P1"
 
+# version = "0.1.1"  # semver "maj.min.patch"
+# license = ""       # e.g., "MIT", "GPL", "Apache 2.0"
+# authors = ["..."]  # e.g., ["Joe Smith (joesmith@noemail.com)", "John Snow (johnsnow@noemail.com)"]
+
 [dependencies]
 
+# For remote import, use the `{ git = "...", subdir = "...", rev = "..." }`.
+# Revision can be a branch, a tag, and a commit hash.
+# MyRemotePackage = { git = "https://some.remote/host.git", subdir = "remote/path", rev = "main" }
+
+# For local dependencies use `local = path`. Path is relative to the package root
+# Local = { local = "../path/to" }
+
+# To resolve a version conflict and force a specific version for dependency
+# override use `override = true`
+# Override = { override = true, local = "../conflicting/version" }
```

Comment:
Being opinionated: I prefer the `override = true` to be at the end of the line, but am fine with this if you prefer it this way.

Reply:
I agree with you, my motivation was that to a reader it's a bit more explicit and easier to understand than look through the line in search of override (especially on super long lines).